style.css

```diff
@@ -66,11 +65,11 @@
 /* BODY SECTION */
 
 /* Body styling */
+
 body {
   width: 381px;
```

Comment:
- [ ] Also, make sure you fix the `background image` to stick to the design as it is required 😃 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-04-30 020656](https://user-images.githubusercontent.com/57408419/235329183-a7f0df13-1671-4a3b-98da-49f4400418d6.png)| |![Screenshot 2023-04-30 020729](https://user-images.githubusercontent.com/57408419/235329187-ffd4fbb5-d65f-47e6-b5bf-0dcef8f61872.png)| |

- [ ] Kindly, make sure you fix the color for technologies, to stick to the design as it is required. 😃, it suppose to be `
#36B37E`. Also `margin` and `font-size` 👍  

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-04-30 020921](https://user-images.githubusercontent.com/57408419/235329217-12c72c6b-52b8-4930-a827-617cb58f8d64.png)| |![Screenshot 2023-04-30 020819](https://user-images.githubusercontent.com/57408419/235329231-844ec39b-436b-4288-9476-dd49a5ce30c4.png)| |